tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,5 +1,46 @@
-{{frost-navigation}}
-
-<div class='dummy-content'>
-  {{outlet}}
+<!-- BEGIN-SNIPPET template -->
+{{frost-navigation
+  logo-container=(component
+    'custom-logo'
```

Comment:
Just a style preference, but I like to see the component helper and name on the same line, i.e.

```
  logo=(component 'custom-logo'...
```